spec/models/book_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+describe Book do
+  subject(:book) { build(:book) }
+
+  it { is_expected.to be_valid }
+  it { is_expected.to validate_presence_of(:genre) }
+  it { is_expected.to validate_presence_of(:author) }
+  it { is_expected.to validate_presence_of(:image) }
+  it { is_expected.to validate_presence_of(:title) }
+  it { is_expected.to validate_presence_of(:editor) }
+  it { is_expected.to validate_presence_of(:year) }
+  it { is_expected.to validate_numericality_of(:year).only_integer }
+  it { is_expected.to validate_length_of(:year).is_equal_to(4) }
+end
+
+describe Book do
```

Comment:
I think that this whole describe is doing the same validation that in the 6th line, because you are validating that should be valid in the 6th line, the part of checking that a new record was added  after creation, should be in the controller, that it's not in the scope for this PR,